utils/v2tofhirr4/mapping_defs.bal

```diff
@@ -74,7 +74,7 @@ public type OrcToImmunization function (Orc orc) returns r4:Immunization;
 # + mshToMessageHeader - MSH segment to MessageHeader FHIR resource mapping function  
 # + pv2ToEncounter - PV2 segment to Encounter FHIR resource mapping function  
 # + orcToImmunization - ORC segment to Immunization FHIR resource mapping function
-public type SegmentToFhir record {
+public type V2SegmentToFhirMapper record {
```

Comment:
Isn't the name of this module `health.hl7v2.utils.v2tofhirr4`?

Do we need to include V2 and Fhir explicitly in the record name, since the module name implies it (assuming there are no other mappers)?

Reply:
There can be other custom mappers in future like HL7 message mappers, data type mappers. But as per the current use cases, segment  mapper is enough. So to explicitely mention the transformation from HL7 segment to FHIR(FHIR resource) we decided to use "V2SegmentToFhirMapper"